external/storm-hdfs/src/main/java/org/apache/storm/hdfs/spout/HdfsSpout.java

```diff
@@ -163,12 +164,19 @@ public HdfsSpout setClocksInSync(boolean clocksInSync) {
     return this;
   }
 
-
+  /**
+	* @deprecated use {@link #setIgnoreSuffix(List)} instead.
+	*/
   public HdfsSpout setIgnoreSuffix(String ignoreSuffix) {
```

Comment:
No needed to deprecate old method and add new one. You just extend the existing method to accept a space separate list of file extensions.  That would preserve compatibility and extend it at the same time. conversion to an array can be done internally.